crates/bevy_dev_tools/Cargo.toml

```diff
@@ -10,14 +10,24 @@ keywords = ["bevy"]
 
 [features]
 bevy_ci_testing = ["serde", "ron"]
+bevy_ui_debug = []
```

Comment:
I'm wondering if I should do all the libs added here optional by feature flag, they might be used in the future by default, but at least now they are optional